uber/templates/summary/csv_import.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base-admin.html" %}
+{% block title %}CSV Import{% endblock %}
+{% block content %}
+
+<h2> CSV Table Imports </h2>
+This attempts to import a csv file for the table you select. <strong>Please do not refresh this page.</strong>
```

Comment:
The only fix for this would be to store the data you want to display in the session and then do the redirect and then grab it back out.  The problem is you can't store things like database model instances in the session, so you'd need to serialize them to dictionaries using ``to_dict`` which would make the refactor kind of annoying.  So it might just not be worth the effort, especially for a feature so few people will use.